src/ontology/mondo-ingest.Makefile

```diff
@@ -541,6 +541,28 @@ $(REPORTDIR)/%.subclass.added.robot.tsv $(REPORTDIR)/%.subclass.confirmed.robot.
 	--mondo-mappings-path $(TMPDIR)/mondo.sssom.tsv \
 	--onto-config-path metadata/$*.yml
 
+##################################
+## Externally managed content ####
+##################################
+
+EXTERNAL_CONTENT_DIR=external
+
+$(TMPDIR)/nord.tsv:
+	wget "https://rdbdev.wpengine.com/wp-content/uploads/mondo-export/rare_export.tsv" -O $@
```

Comment:
How are versions of downloaded content managed between one run to the next, e.g. appended with the download date or moved to an archive type folder and the date added then?

Do the downloaded files always contain all content or are only new and changed content?

Reply:
We can take this offline, added to meeting notes.

Reply:
Actually this is a good point, and I can easily add that. I will ping you once it is done. It is achieved by tagging the robot template owl version with the downloaded date.

Reply:
As you can see below, I now added explicit versioning information at least to the OWL file:

```
$(EXTERNAL_CONTENT_DIR)/%.robot.owl: $(EXTERNAL_CONTENT_DIR)/%.robot.tsv
	$(ROBOT) template --template $< \
	 	annotate \
				--ontology-iri $(URIBASE)/mondo/external/nord.robot.owl \
				--version-iri $(URIBASE)/mondo/external/$(TODAY)/nord.robot.owl \
				-o $@
```

This looks like:

```
        <owl:versionIRI rdf:resource="http://purl.obolibrary.org/obo/mondo/external/2024-02-22/nord.robot.owl"/>
```

The cool thing is that we can make this work like a PURL, and it tells us when it was created!

The "archiving" strategy emerges than naturally:

When we make a data refresh, we create a GitHub release. The GitHub release creates a `tag`, e.g. `v2024-02-22`. That `tag` can be used to access the datafile (archiving strategy). This is cool because it is exactly how it is done across all of OBO! 

If you have more questions about this, add to meeting minutes and we can debate! :)